lib/utils/funding.js

```diff
@@ -62,6 +60,14 @@ function getFundingInfo (idealTree, opts) {
     )
   }
 
+  function retrieveFunding (funding) {
+    return typeof funding === 'string'
+      ? {
+        url: funding
```

Comment:
nit: I would change this syntax to a one liner; `{ url: funding }`. When I read this now it makes me think it's a block scope and something _interesting_ is happening.